triage/triaging.rst

```diff
@@ -7,32 +7,24 @@ Triaging an Issue
 This section of the devguide documents the :ref:`issue tracker <tracker>` for
 users and developers.
 
-Contributors with the Triager role on the issue tracker can triage issues
-directly without any assistance.
-
 
 Checklist for Triaging
 ======================
 
-* Read the issue comment(s).
-* Review and set classification fields
-    - Title: should be concise with specifics which are helpful to someone
-      scanning a list of issue titles. (Optional, if possible) Add a
-      prefix at the start of the title to indicate the module, e.g. IDLE,
-      doc, or async.
-    - Type
-    - Stage
-    - Components: multiple items may be set
-    - Versions: set if known, leave blank if unsure. Multiple items may be set.
-* Review and set process fields
-    - Status
-    - Superseder
-    - Assignees
-    - Nosy List
-    - Priority
-    - Keywords
-* (Optional) Leave a brief comment about the proposed next action needed. If
-  there is a long message list, a summary can be very helpful.
+* Check that the title is concise with specifics which are helpful to
+  someone scanning a list of issue titles.
+* For pull request, ensure that the corresponding issue is added before
+  the title (``gh-NNNNN: ...``).
+* Read the initial message and the comments.
```

Comment:
Shouldn't this go first, as it was before? Otherwise, traigers can't know if the issue title is accurate and descriptive about the actual issue involved...

Reply:
Done